### ISSUE TITLE
Save & Discard buttons areas are disproportioned

### ISSUE THREAD

**Issue by LeonardoAcquaroli** [ox-vgg/via#21]
Hi,
I lost hours of annotations because of what I think is a very easy-to-fix issue.
The space covered by the button Discard, that opens up when the page asks to collect the cached data, starts right after the "Save" writing. Thus the space in the middle triggers the Discard button which is a very bad practice since Discard is an irreversible action.

<img width="422" alt="image" src="https://github.com/ox-vgg/via/assets/89029338/172784ca-5ba3-4538-81fe-82bf253e84de">

Thanks for the attention.
P.s. I'm complaining about this little thing but the project overall is great!

**Comment by thelinuxmaniac** [comment]
Thank you @LeonardoAcquaroli for sharing your report. I am sorry for your data loss caused by this poor design choice.

Since we the development of VIA happens in gitlab (with github acting as mirror), I have filed an [issue in gitlab](https://gitlab.com/vgg/via/-/issues/451). We will address this issue and share updates.

**Comment by thelinuxmaniac** [comment]
Use `margin` (not `padding`) to add the desired space between two buttons. Issue fixed in 64b3b47abab2a56e55d80263f1aaed3c6835f8bb

```
-    hstr.push('<a style="padding-left:2em; cursor: pointer; color: blue;" onclick="remove_via_data_from_localStorage(); show_home_panel();" title="Discard annotation data">Discard</a>');
+    hstr.push('<a style="margin-left:2em; cursor: pointer; color: blue;" onclick="remove_via_data_from_localStorage(); show_home_panel();" title="Discard annotation data">Discard</a>');
```

This fix is available in the following [via.html.zip](https://github.com/user-attachments/files/15835076/via.html.zip)

**Comment by LeonardoAcquaroli** [comment]
Great, thank you!
Super efficient
